python/lsst/cp/pipe/ptc/cpSolvePtcTask.py

```diff
@@ -260,6 +289,21 @@ def run(self, inputCovariances, camera=None, detId=0):
                 means, variances, and exposure times
                 (`lsst.ip.isr.PhotonTransferCurveDataset`).
         """
+        # Check parameters
+        fitMatrixSide = self.config.maximumRangeCovariancesAstierFullCovFit
+        measureMatrixSide = self.config.maximumRangeCovariancesAstier
+        if self.config.ptcFitType == "FULLCOVARIANCE":
+            if fitMatrixSide > measureMatrixSide:
+                raise RuntimeError("Covariance fit size %s is larger than"
+                                   "measurement size %s.",
+                                   fitMatrixSide, measureMatrixSide)
+            if self.config.doSubtractLongRangeCovariances:
```

Comment:
If this only makes sense for the FULLCOVARIANCE model, then that can be done in the config validate as well.